SECRUITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+The Defence Digital Digital Standards Policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.
+
+## Reporting a Vulnerability
+
+If you believe you have found a security vulnerability, please submit your report to us using [here](#)
```

Comment:
We have two options here do we open a HackerOne account for the org or do we sent to a email address